cmd/tendermint/commands/show_validator.go

```diff
@@ -5,31 +5,55 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"github.com/tendermint/tendermint/crypto"
 	tmjson "github.com/tendermint/tendermint/libs/json"
+	tmnet "github.com/tendermint/tendermint/libs/net"
 	tmos "github.com/tendermint/tendermint/libs/os"
 	"github.com/tendermint/tendermint/privval"
+	tmgrpc "github.com/tendermint/tendermint/privval/grpc"
 )
 
 // ShowValidatorCmd adds capabilities for showing the validator info.
 var ShowValidatorCmd = &cobra.Command{
-	Use:     "show-validator",
+	Use:     "show-validator [chainID]",
 	Aliases: []string{"show_validator"},
 	Short:   "Show this node's validator info",
+	Args:    cobra.ExactArgs(1),
```

Comment:
I believe this means that the user has to add a chainID even if it's not needed (in the case of a "non grpc" privval). I don't know if we want to consider just making the argument optional or using a flag instead (I thought that config already had chainID within it).

Reply:
I think this makes it a breaking change to the CLI

Reply:
Removed the need for the arg by loading it from genesis. No longer breaking 😄